packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/PairedResultGraph/BasicPairedResultGraphs/SimplePairedResult.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '@cerner/terra-graphs-docs/lib/Css/ExampleGraphContainer.module.scss';
+import '@cerner/terra-graphs/lib/components/Graph.module.scss';
+import '@cerner/terra-graphs/lib/components/PairedResult/PairedResultGraph.module.scss'; import pairedResultDefault from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/PairedResult/pairedResultDefault';
```

Comment:
This looks like it needs a line break.

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/186/commits/e5403ae8e7205354f696f6c9c3348050074cb7ac).